### PR TITLE
get/putspi

### PR DESCRIPTION
**Pull request by mklight**
Signed-off-by: Matt K. Light <mklight@us.ibm.com>

### CODE REVIEW

File: ecmd-core/capi/ecmdClientCapi.H [hunk 1]
```diff
@@ -5282,6 +5282,51 @@ uint32_t ecmdI2cWriteOffsetHidden(ecmdChipTarget & i_target, uint32_t i_engineId
 uint32_t ecmdQueryFileLocationHidden2(const ecmdChipTarget & i_target, ecmdFileType_t i_fileType, std::list<ecmdFileLocation> & o_fileLocations, std::string & io_version);
 //@}
 
+/** @name SPI Functions */
+//@{
+#ifndef ECMD_REMOVE_SPI_FUNCTIONS
+/**
+ @brief Read data from a SPI device at the given address
+ @param i_target Struct that specifies the target to operate on (see target depth and states below)
+ @param i_engineId SPI engine to use
```
Comment (kahnevan) [inline ecmd-core/capi/ecmdClientCapi.H]: i_engine

File: ecmd-core/capi/ecmdClientCapi.H [hunk 2]
```diff
@@ -5282,6 +5282,51 @@ uint32_t ecmdI2cWriteOffsetHidden(ecmdChipTarget & i_target, uint32_t i_engineId
...
+
+ TARGET DEPTH  : pos<br>
+ TARGET STATES : Unused <br>
+*/
+uint32_t spiRead(const ecmdChipTarget & i_target, uint32_t i_engine, uint32_t i_select, uint64_t i_address, uint32_t i_bytes, uint32_t i_mode, ecmdDataBuffer & o_data);
+
+/**
+ @brief Write the provided data into the SPI device at the given address
+ @param i_target Struct that specifies the target to operate on (see target depth and states below)
+ @param i_engineId SPI engine to use
```
Comment (kahnevan) [inline ecmd-core/capi/ecmdClientCapi.H]: i_engine

File: ecmd-core/capi/ecmdClientCapi.H (hunk 1 above)
Comment (mklight) [reply ecmd-core/capi/ecmdClientCapi.H]: done

File: ecmd-core/capi/ecmdClientCapi.H (hunk 2 above)
Comment (mklight) [reply ecmd-core/capi/ecmdClientCapi.H]: done

**Pull request merged**